Description: Attempt to disable lld linker to use gold instead
 .
 chromium-browser (84.0.4147.30-0ubuntu1~ppa1~20.04.1) focal; urgency=low
 .
Author: Brad Harper <bjharper@gmail.com>

---
--- chromium-browser-84.0.4147.30.orig/build/config/compiler/BUILD.gn
+++ chromium-browser-84.0.4147.30/build/config/compiler/BUILD.gn
@@ -392,7 +392,7 @@ config("compiler") {
   # Linux-specific compiler flags setup.
   # ------------------------------------
   if ((is_posix || is_fuchsia) && use_lld) {
-    ldflags += [ "-fuse-ld=lld" ]
+    # ldflags += [ "-fuse-ld=lld" ]
     if (current_cpu == "arm64") {
       # Reduce the page size from 65536 in order to reduce binary size slightly
       # by shrinking the alignment gap between segments. This also causes all
